contrib/spec/podman.spec.in

```diff
@@ -48,7 +48,7 @@ Epoch: 99
 %else
 Epoch: 0
 %endif
-Version: 1.8.2
+Version: 1.8.3
```

Comment:
shouldn't this be 1.8.3-dev?

Reply:
Not for the RPM, we don't append `-rc` there